k2/python/csrc/torch/ragged.cu

```diff
@@ -122,11 +142,28 @@ static void PybindRaggedTpl(py::module &m, const char *name) {
         return std::make_pair(ans, value_indexes_tensor);
       },
       py::arg("src"), py::arg("indexes"), py::arg("need_value_indexes") = true);
+
+  m.def(
+      "index_ragged_with_ragged_int",
```

Comment:
I would suggest renaming `index_ragged_with_ragged_int` and `index_tensor_with_ragged_int` to `index`.

Pybind11 supports function name overloading.

Reply:
Ah, OK, will do later (but maybe later, I just added the TODO in case I forget this but have something else to do now..)